Sources/ShellOut.swift

```diff
@@ -23,81 +24,114 @@ import Foundation
  */
 @discardableResult public func shellOut(to command: String,
                                         arguments: [String] = [],
-                                        at path: String = ".") throws -> String {
+                                        at path: String = ".",
+                                        printOutput: Bool = false) throws -> String {
     let process = Process()
     let command = "cd \"\(path)\" && \(command) \(arguments.joined(separator: " "))"
-    return try process.launchBash(with: command)
+    return try process.launchBash(with: command, printOutput: printOutput)
 }
 
 /**
  *  Run a series of shell commands using Bash
  *
  *  - parameter commands: The commands to run
  *  - parameter path: The path to execute the commands at (defaults to current folder)
+ *  - parameter printOutput: Whether `STDERR` and `STDOUT` should be printed ot not.
  *
  *  - returns: The output of running the command
  *  - throws: `ShellOutError` in case the command couldn't be performed, or it returned an error
  *
  *  Use this function to "shell out" in a Swift script or command line tool
  *  For example: `shellOut(to: ["mkdir NewFolder", "cd NewFolder"], at: "~/CurrentFolder")`
  */
-@discardableResult public func shellOut(to commands: [String], at path: String = ".") throws -> String {
+@discardableResult public func shellOut(to commands: [String], at path: String = ".", printOutput: Bool = false) throws -> String {
     let command = commands.joined(separator: " && ")
-    return try shellOut(to: command, at: path)
+    return try shellOut(to: command, at: path, printOutput: printOutput)
 }
 
 // Error type thrown by the `shellOut()` function, in case the given command failed
 public struct ShellOutError: Swift.Error {
+    /// The buffer data retuned by `STDERR`.
+    public let stderr: Data
```

Comment:
But `string` is so much easier to consume, especially since this package is mostly targeted towards scripting, where convenience is key.